functions-methods.js

```diff
@@ -10,6 +10,22 @@
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
 
 
+function getEmailDomain(emailAddress){
+    return emailAddress.match(/@([^@]+)/);
+//     Dit moest ik opzoeken. Blijkbaar kun je gebruik maken van een regular expression voor het vinden van een match in een String, die daadwerkelijk ook een String teruggeeft. De schrijfwijze hiervan is een beetje funky, maar komt op het volgende neer:
+//     @: Zoekt het apenstaartje in het e-mailadres op en zegt: begin hier met matchen.
+//     ([^.]+): Dit is de capture group en zegt: neem alles wat matcht met wat hierbinnen staat
+//     [^.]+: [ en ] definieren de lijst karakters waar we op checken
+//     '^' betekent: NIET deze karakters
+//     '@' betekent: we nemen alles TOT de @ (Die er niet nog eens in staat)
+//     '+' betekent: match op 1 of meer karakters
+}
```

Comment:
Keurig omschreven hoe regex werkt, erg handig om in je arsenaal te hebben. Maar hoe zou je het hebben gedaan zonder regex? Dat is meer het doel van de opdracht namelijk.